### DISCUSSION TITLE
Need help to import ViTTiny16

### DISCUSSION THREAD

**Discussion by tanjummotin** [keras-team/keras-cv#2156] [Q&A]
ImportError: cannot import name 'ViTTiny16' from 'keras_cv.models' (/usr/local/lib/python3.10/dist-packages/keras_cv/models/__init__.py)
I am getting this error. Can someone help me?Is this module now not availabe?

**Comment by sampathweb** [comment]
These are legacy models that need to be the new API.  You can access them via `from keras_cv.src.models.legacy import ViTTiny16` for now, but we will be porting some of them to align with the supported models in `keras_cv.models` - https://keras.io/api/keras_cv/models/
